app/Http/Resources/V1/CurrikiGo/LmsSettingCollection.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Resources\V1\CurrikiGo;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class LmsSettingCollection extends ResourceCollection
+{
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return ['data' => $this->collection];
```

Comment:
Please specify what is the response data.
Not just use 'data'. I guess it would be 'settings'?